charactersheet/charactersheet/models/dm/encounter_sections/player_text.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
+
+function PlayerText() {
+    var self = this;
+    self.ps = PersistenceService.register(PlayerText, self);
+    self.mapping = {
+        include: ['characterId', 'encounterId', 'description']
+    };
+
+    self.characterId = ko.observable();
+    self.encounterId = ko.observable();
+    self.description = ko.observable();
+
+    //Public Methods
+
+    self.clear = function() {
+        var values = new PlayerText().exportValues();
+        var mapping = ko.mapping.autoignore(self, self.mapping);
+        ko.mapping.fromJS(values, mapping, self);
+    };
+
+    self.importValues = function(values) {
+        var mapping = ko.mapping.autoignore(self, self.mapping);
+        ko.mapping.fromJS(values, mapping, self);
+    };
+
+    self.exportValues = function() {
+        var mapping = ko.mapping.autoignore(self, self.mapping);
+        return ko.mapping.toJS(self, mapping);
+    };
+
+    self.save = function() {
+        self.ps.save();
+    };
+
+    self.delete = function() {
+        self.ps.delete();
+    };
+
+    // UI Methods
+
+    self.longDescription = ko.pureComputed(function() {
+        return self._plainTextDescription().substr(0, 200).trim() + '...';
+    });
+
+    self.shortDescription = ko.pureComputed(function() {
+        return self._plainTextDescription().substr(0, 100).trim() + '...';
+    });
+
+    // Private Methods
+
+    /**
+     * Returns the same text as the description, but without HTML/MD artifacts.
+     */
+    self._plainTextDescription = ko.pureComputed(function() {
```

Comment:
It seems like we use this, and will continue to use this, in various places. Do you think it would be good to make it a common utility?